packages/quantic/cypress/e2e/default-2/generatedAnswer/generated-answer.cypress.ts

```diff
@@ -661,7 +661,8 @@ describe('quantic-generated-answer', () => {
                 visitGeneratedAnswer({useCase: param.useCase});
               });
 
-              it('should properly display the like button', () => {
+              //Skipping this test for now, it will be adapted to the new logic in another ticket
```

Comment:
Created the ticket to adapt this:
```suggestion
              // TODO: SFINT-5608 Skipping this test for now, it will be adapted to the new logic in another ticket
```